### ISSUE TITLE
Rule Change: Sort members by original exported name, not by renamed import name

### ISSUE THREAD

**Issue by j-murata** [eslint/eslint#18017]
### What rule do you want to change?

`sort-imports`

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
import {
  a,
  b as myAliasName,
  c,
} from 'some-module';
```

### What does the rule currently do for this code?

If `{ "ignoreMemberSort": false }` is set in this rule, it will result in an error unless reordered as follows:

```js
import {
  a,
  c,
  b as myAliasName,
} from 'some-module';
```

### What will the rule do after it's changed?

If there is an option to have import members sorted by their original exported name, then even if the alias name is changed, there is no need to reorder them.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

In the process of source code refactoring, it is often necessary to change the alias name of imports. In such cases, it is troublesome to change the import order.

**Comment by mdjermanovic** [comment]
Hi @j-murata, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates) so we are not adding new options to the `sort-imports` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.

**Comment by j-murata** [comment]
Hi @mdjermanovic, thanks for the reply, I have one question.

Is this `sort-imports` included in the _formatting_ and _stylistic_ rules?
I don't see it in the lists below, how do I know which rules are subject to the freeze or not?
* [Deprecation of formatting rules » The deprecated rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules/#the-deprecated-rules)
* [ESLint Stylistic » Rules](https://eslint.style/packages/default#rules)
